### PR TITLE
Remove Subscription Change Options

### PR DESCRIPTION
**Pull request by Luke9389**
### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/259131

# Tests / Web QA
1. Sign into any account
2. Go to Settings > Policies > Group
3. Make a new Collect Policy
4. Toggle Auto renew Off
5. Verify that the modal looks like this:
Screenshot 2023-01-31 at 4 21 04 PM

### CODE REVIEW

File: lib/CONST.jsx [hunk 1]
```diff
@@ -819,21 +819,11 @@ export const CONST = {
...
-        PREFER_MANUAL_RENEW: {
-            id: 'preferManualRenew',
-            label: 'Prefer to manually renew subscription',
-            prompt: 'What makes you consider not renewing with Expensify?',
-        },
         BUSINESS_CLOSING: {
             id: 'businessClosing',
             label: 'Company closing, downsizing, or acquired',
             prompt: 'What software are you migrating to and what led to this decision?',
         },
```
Comment (MariaHCD) [inline lib/CONST.jsx:826]:
```suggestion
        BUSINESS_CLOSING: {
            id: 'businessClosing',
            label: 'Company closing, downsizing, or acquired',
            prompt: 'Would you have liked to continue using Expensify, or would you recommend it in your network?',
        },
```

File: lib/CONST.jsx [hunk 2]
```diff
@@ -819,21 +819,11 @@ export const CONST = {
             label: 'Inadequate customer support',
             prompt: 'What software are you migrating to and what led to this decision?',
```
Comment (MariaHCD) [inline lib/CONST.jsx:820]:
```suggestion
            prompt: 'What about our customer support didn\'t work well for you?',
```

**Review by MariaHCD — CHANGES_REQUESTED** [review]
Wasn't able to comment on the unchanged lines but the prompts need to be updated for all these options as per the issue :)

**Comment by Luke9389** [comment]
Oh I missed that. Thanks @MariaHCD

**Review by MariaHCD — APPROVED** [review]
LGTM!

[Expensify - Settings - Policies.webm](https://user-images.githubusercontent.com/12268372/216271037-052ee3d0-2518-4abe-b92a-93b07fb17e0c.webm)

**Comment by flaviadefaria** [comment]
Shoot I'm sorry I added the correct copy in one place but not in the other this is the correct copy:

id: 'tooLimited' - update prompt to "What software are you migrating to and what led to this decision?"
id: 'tooExpensive' - update prompt to "What software are you migrating to and what do you think is a fair price?"
id: 'inadequateSupport', - update prompt to "What software are you migrating to and what led to this decision?"
id: 'businessClosing' - update prompt to "What software are you migrating to and what led to this decision?"

Basically the prompt should be the same in all cases.

**Comment by MariaHCD** [comment]
Ah, whoops. @flaviadefaria, so to clarify all the prompts should be `What software are you migrating to and what led to this decision?`?

**Comment by MariaHCD** [comment]
Created a PR fix here: https://github.com/Expensify/expensify-common/pull/495

**Comment by flaviadefaria** [comment]
Thanks @MariaHCD! Also `TOO_LIMITED` should have the label shortened to 'Functionality needs improvement'
